Gaijinizer patch/Map404.txt

```diff
@@ -1,238 +1,296 @@
-
+> CONTEXT: Map404/events/1/pages/0/101/Dialogue/enemy29
+(I had thought that individuals from that world
+  who had accepted intervention were not
+  uncommon, and that talk of "singularities" was
+  exaggeration. But now I see, certainly...)#RonIffy
 > END STRING
 
 > BEGIN STRING
 \\N[1]は、装備品とアイテムを回収した
-> CONTEXT: Map404/events/1/pages/0/108/Dialogue < UNTRANSLATED
-
+> CONTEXT: Map404/events/1/pages/0/108/Dialogue
+\\N[1] has recovered his weapons and items.#Ron
 > END STRING
 
 > BEGIN STRING
 これでよし…と
-> CONTEXT: Map404/events/1/pages/0/119/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map404/events/1/pages/0/119/Dialogue/kazuya
+That should be everything.#Ron
 > END STRING
 
 > BEGIN STRING
 いやぁ、まいったなぁ
 
 服と装備を身につけた瞬間に襲う気だったのに、
 そこまで読まれてたら、やる気が削がれちゃった♪
-> CONTEXT: Map404/events/1/pages/0/122/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map404/events/1/pages/0/122/Dialogue/enemy29
+Haah, you know I thought about attacking you the
+moment you put your equipment and clothes back
+on, but after you read me so well, I lost all my
+motivation. ♪#Ron
 > END STRING
 
 > BEGIN STRING
 それに、いくら確信があるとはいえ
 悪魔と取引するときは、ちゃんと約束させないと…
 
 猫も悪魔も、気まぐれを起こすのは唐突だよ？
-> CONTEXT: Map404/events/1/pages/0/127/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map404/events/1/pages/0/127/Dialogue/enemy29
+And while you may have had a lot of faith in me,
+if you don't get a promise from a demon when
+doing a deal, whether demon or cat, we can
+suddenly turn on you on a whim.#RonIffyWording
 > END STRING
 
 > BEGIN STRING
 …………じゃあ、何でお前は起こさなかったんだよ
-> CONTEXT: Map404/events/1/pages/0/132/Dialogue/other2 < UNTRANSLATED
-
+> CONTEXT: Map404/events/1/pages/0/132/Dialogue/other2
+So then why didn't you turn on him?#Ron
 > END STRING
 
 > BEGIN STRING
 うーん……
 
 ボクも\\N[1]くんのレディーキラーっぷりに
 やられちゃったのかもしれないねぇ♡
-> CONTEXT: Map404/events/1/pages/0/134/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: Map404/events/1/pages/0/134/Dialogue/enemy29
+Well...
 
+I guess \\N[1]-kun's natural ladykiller
+charms might've got to me, too... ♡#Ron
 > END STRING
 
 > BEGIN STRING
 …………なんだそりゃ
-> CONTEXT: Map404/events/1/pages/0/139/Dialogue/other2 < UNTRANSLATED
-
+> CONTEXT: Map404/events/1/pages/0/139/Dialogue/other2
+What kind of answer is that?#Ron
 > END STRING
 
 > BEGIN STRING
 あ、\\N[1]くん。分かってるとは思うけど
 
 小さくなった時の影響で
 \\C[2]装備していた武器や装飾品はすべて外れてる。
-> CONTEXT: Map404/events/1/pages/0/142/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map404/events/1/pages/0/142/Dialogue/enemy29
+Oh yeah, I'm sure you know this already,
+\\N[1]-kun, but all of your equipment was
+unequipped when you were shrunk.#Ron
 > END STRING
 
 > BEGIN STRING
 だから装備品は、もう一度装備し直す必要がある
 
 特に、これから会うティターニアとは
 \\C[2]穏便に事が済みそうな気配はしないからね
-> CONTEXT: Map404/events/1/pages/0/147/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map404/events/1/pages/0/147/Dialogue/enemy29
+So you'll have to re-equip everything. Especially
+since you're going to see Titania, and I don't
+think your meeting's going to go smoothly.#Ron
 > END STRING
```

Comment:
Some rewording here:
```
You'll need to re-equip everything.
Especially if you're going to see Titania, 
\\C[2]because I don't think your meeting will go
smoothly.